Makefile

```diff
@@ -23,12 +21,28 @@ for line in sys.stdin:
 		print("%-20s %s" % (target, help))
 endef
 export PRINT_HELP_PYSCRIPT
-
 BROWSER := python -c "$$BROWSER_PYSCRIPT"
 
 help:
 	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
+init:
+	@python3 -m venv .enterprise_extensions --prompt enterprise_extensions
+	@./.enterprise_extensions/bin/python3 -m pip install -U pip setuptools wheel
+	@./.enterprise_extensions/bin/python3 -m pip install -r requirements.txt -U
+	@./.enterprise_extensions/bin/python3 -m pip install -r requirements_dev.txt -U
+	@./.enterprise_extensions/bin/python3 -m pre_commit install --install-hooks --overwrite
```

Comment:
Since you aren't doing linting or formatting you don't need this.

Reply:
Yeah, I was leaving those in ad I wanted to try and convince folks to use them, but I can add them back in if I do :-)